module/Activity/view/activity/activity/view.phtml

```diff
@@ -144,4 +144,9 @@ $this->headTitle($this->translate('Activities')); ?>
         echo $this->formSubmit($submit);
         echo $this->form()->closeTag();?>
     <?php endif; ?>
+
+    <?php if ($this->acl('activity_acl')->isAllowed('activity', 'create')): ?>
```

Comment:
This is outside the scope for this pull requests, and is something for a new issue, that can be resolved later